### DISCUSSION TITLE
Regex for jitsi meet

### DISCUSSION THREAD

**Discussion by jokay** [renovatebot/renovate#11182] [Request Help]
How to support [Jitsi Meet](https://jitsi.org/jitsi-meet/) updates?

The tags they use have a number only, see [here](https://hub.docker.com/r/jitsi/web/tags?page=1&ordering=last_updated&name=stable-5), e.g.

- `stable-5765`
- `stable-5765-1`

May something like this work?

```json
{
      "packageNames": [
        "docker.io/jitsi/web"
      ],
      "versionScheme": "regex:^stable-\\.?(?<major>\\d+)(?<minor>\\d+)?$"
},
```

**Comment by rarkins** [comment]
You could also try loose versioning if all you care about is sorting

**Reply by jokay** [reply]
@rarkins tried with **renovate/renovate:slim** version **26.19.1** and the following config without success.

```yml
{
  "packageNames": [
    "docker.io/jitsi/web",
    "docker.io/jitsi/prosody",
    "docker.io/jitsi/jicofo",
    "docker.io/jitsi/jvb"
  ],
  "versioning": "loose",
  "allowedVersions": "/^stable-\\d+$/"
}
```

Any idea what might be wrong to get it updated from e.g. `docker.io/jitsi/web:stable-5963` to `docker.io/jitsi/web:stable-6173`?

**Reply by jokay** [reply]
And optional `"/^stable-\\d+-\\d+$/"` to have something like `stable-5390-3` work as well?

**Reply by jokay** [reply]
@viceice or may be you can help with this?

**Reply by jokay** [reply]
```sh
DEBUG: Dependency docker.io/jitsi/web has unsupported value stable-5963 (repository=demo.app)
DEBUG: getDigest(https://index.docker.io, jitsi/web, stable-5963) (repository=demo.app)
```

**Reply by viceice** [reply]
I would prefer regex versioning over loose, as loose would would sort `stable-10000` lower than `stable-9999`

```json
{
  "packageNames": [
    "docker.io/jitsi/web",
    "docker.io/jitsi/prosody",
    "docker.io/jitsi/jicofo",
    "docker.io/jitsi/jvb"
  ],
  "versioning": "regex:^stable-(?<major>\\d+)(?:-(?<minor>\\d+))?$"
}
```

I think this should do it.

Check the `packagefiles with updates` debug log message for `depName` to match against.

**Reply by jokay** [reply]
Thx, I think `versioning` should be `versionScheme` in this case?

**Reply by viceice** [reply]
Nope, versioningScheme is outdated and no longer supported

**Reply by jokay** [reply]
> Nope, versioningScheme is outdated and no longer supported

Good to know ^^

**Reply by viceice** [reply]
i think `versioningScheme` was never supported. we don't have any migration for it. So i think your first config didn't work, because of wrong property name. 😏 https://docs.renovatebot.com/configuration-options/#versioning

**Reply by jokay** [reply]
Well, this is what worked yesterday to update to `stable-6173`:

```yml
{
    "packageNames": [
        "docker.io/jitsi/web",
        "docker.io/jitsi/prosody",
        "docker.io/jitsi/jicofo",
        "docker.io/jitsi/jvb"
    ],
    "versionScheme": "regex:^stable-?(?<major>\\d+)(?<minor>-\\d+)?$"
}
```

So no idea how, but it did :wink:

**Reply by jokay** [reply]
But I updated to `versioning` and your regex now to be sure to use the correct one.

**Reply by viceice** [reply]
I don't think it applied your `versionScheme`, i think some other versioning allowed it to work, yoou need to search the docs to check what's happened if you like to know it.
